src/index.tsx

```diff
@@ -1,28 +1,28 @@
 // Individual commands
-import { boldCommand } from "./commands/markdown-commands/boldCommand";
-import { italicCommand } from "./commands/markdown-commands/italicCommand";
-import { strikethroughCommand } from "./commands/markdown-commands/strikethroughCommand";
-import { linkCommand } from "./commands/markdown-commands/linkCommand";
-import { quoteCommand } from "./commands/markdown-commands/quoteCommand";
-import { imageCommand } from "./commands/markdown-commands/imageCommand";
-import { CommandController } from "./commands/command-controller";
-import type { TextController } from "./types/CommandOptions";
-import { TextAreaTextController } from "./text/textarea-text-controller";
+import {boldCommand} from "./commands/markdown-commands/boldCommand";
```

Comment:
It looks like the code formatting does not follow the `.prettierrc` config file.

`bracketSpacing` is true by default.